boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -25,7 +25,8 @@ const struct State *App_GetDriveState(void)
     static struct State drive_state = {
         .name         = "DRIVE",
         .run_on_entry = DriveStateRunOnEntry,
-        .run_on_tick  = DriveStateRunOnTick,
+        .run_on_tick_1Hz  = DriveStateRunOnTick,
```

Comment:
I wanna say it would make sense to have separate on entry and on exit for 1hz and 1khz?

Reply:
I don't think so, can you think of any examples?

Reply:
Hmm don't have a concrete example for now so I guess we can leave it out (should be easy to extend later on as the need arises)